includes/Elements/Separator/Separator.php

```diff
@@ -235,13 +236,8 @@ public function options( $options ) {
 				'description' => __( 'Choose the desired margin for this element.', 'zionbuilder' ),
 				'min'         => -99999,
 				'max'         => 99999,
-				'sync'        => '_styles.wrapper.styles.default.default',
+				'sync'        => '_styles.wrapper.styles.%%RESPONSIVE_DEVICE%%.default',
```

Comment:
please add "show_responsive_buttons" => true so we can also show the responsive buttons inline